keps/sig-scheduling/5055-dra-device-taints-and-tolerations/README.md

```diff
@@ -203,8 +202,8 @@ unique ID. Both are supported, which creates additional complexity.
 Without a kubectl extension similar to `kubectl taint nodes`, the user
 experience for admins will be a bit challenging. They need to decide how to
 identify the device (by name or with a CEL expression), manually create a
-ResourceSlicePatch with a unique name, then remember to remove that
-ResourceSlicePatch again. For beta, support in `kubectl` for common
+DeviceTaintRule with a unique name, then remember to remove that
```

Comment:
Unique across whole cluster, unique for the node on which the device resides, or unique to the device(s) matching its selection criteria?

(Taints that the classic DRA Intel GPU driver supported earlier, were in per-device sections of per-node CRs.)

Reply:
DeviceTaintRules are cluster-scoped objects, so their name has to be unique in the entire cluster.

Reply:
Ok, it means that *both* the name & the selector for the taint rule, both need to include at least node and device identifiers to avoid such conflicts, e.g. in automated tainting.